core/lib/instrument.c

```diff
@@ -4881,6 +4881,20 @@ dr_raw_tls_cfree(uint offset, uint num_slots)
     return os_tls_cfree(offset, num_slots);
 }
 
+DR_API
+void
+dr_raw_write_tls_slot(reg_id_t *tls_register, ushort offset, void *value)
```

Comment:
The name is not consistent: the idea is that this is "raw TLS".  See all the existing names: they have "raw_tls" in them.  This is not a "raw write": this is a write to a particular type of TLS, "raw TLS".

Reply:
Changed into dr_read_raw_tls_slot(), dr_write_raw_tls_slot() (vs. the _insert versions)